### DISCUSSION TITLE
The output file seems to lack a coordinate system?

### DISCUSSION THREAD

**Discussion by FuYingWang** [NCAR/noahmp#157] [Q&A] [answered]
Hello, I completed a simple simulation, but the 'LDASOUT_DOMAIN1' files seem to have no coordinates. 
For example,  I use Panoply software opening '2023010600.LDASOUT_DOMAIN1' variable 'LAI' (Figure 1)
![image](https://github.com/user-attachments/assets/2a742d9a-6ba3-4b66-bba4-0669d8f470eb)
I can't create image with 'Longitude-Latitude'. I can use 'south_north & west_east'.
Also, I use python read the file (Figure 2), there are no coordinates.
![image](https://github.com/user-attachments/assets/5d8af0b0-2ac1-4bb2-907a-575be6a15eb1)

However, for the 'geo_em.d01.nc' file ( create from WPS ). It can obviously find coordinates (figure 3,4).
![image](https://github.com/user-attachments/assets/1187c8ae-a63d-43fe-815a-a69e8e3a1cee)
![image](https://github.com/user-attachments/assets/641f15c9-23a1-4128-b410-22201c89faf8)

**Comment by cenlinhe** [comment, marked as answer]
Hi, the output file does not include lat,lon coordinate information. You will need to use the coordinate variables from the HRLDAS_setup file or geo_em file.

**Reply by FuYingWang** [reply]
Thanks a lot, and is it possible to assign the geo_ em file coordinate system to the output file using wrf python or NCL? Or other tools?

**Reply by cenlinhe** [reply]
Not sure I understand your question correctly, but there are two ways to do it:
1. read in the coordinate system (XLAT, XLONG) from geo_em file in your data processing script together with your reading of output variables from output files;
2. Use nco command and bash script to add XLAT and XLONG variables directly to the HRLDAS output netcdf file: https://stackoverflow.com/questions/68635806/how-to-add-variable-from-nc1-to-nc2-file-without-deleting-removing-the-variables
